CodeConverter/CodeConverter.csproj

```diff
@@ -41,6 +41,10 @@
     <PackageReference Include="Microsoft.VisualBasic" Version="10.3.0" />
     <PackageReference Include="Microsoft.VisualStudio.Composition" Version="16.9.20" />
     <PackageReference Include="Microsoft.VisualStudio.Threading" Version="16.10.56" />
+    <PackageReference Include="Nullable" Version="1.3.0">
```

Comment:
Needed for ```[NotNullWhen(true)]``` attribute used in [ITypeSymbolExtensions.cs](https://github.com/icsharpcode/CodeConverter/pull/872/files#diff-31724a2fbcd62d20f9b99a2bae23100b7d2693508252015421254edc776fa897)